pkg/api/server/register_images.go

```diff
@@ -10,32 +10,652 @@ import (
 )
 
 func (s *APIServer) registerImagesHandlers(r *mux.Router) error {
+	// swagger:operation POST /images/create images createImage
+	//
+	// Create an image from an image
+	//
+	// ---
+	// produces:
+	// - application/json
+	// parameters:
+	//  - in: query
+	//    name: fromImage
+	//    type: string
+	//    description: needs description
+	//  - in: query
+	//    name: tag
+	//    type: string
+	//    description: needs description
```

Comment:
I suspect you also need a https://swagger.io/docs/specification/2-0/describing-request-body/ to complete this operation